app/components/blacklight/facet_item_component.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
```

Comment:
Moving this component into the same path as the upstream one, so that we can override its behavior globally.

Reply:
Ok, after much git traversing, I have found the original ticket! See https://github.com/pulibrary/orangelight/issues/210. If I'm reading it correctly (and it's 7:15 am my time, so I might not be), it sounds like it was originally added for accessibility, and I suspect we have better ways to do that now, that are hopefully in the Blacklight facet? And if not maybe we can add them there?

Reply:
Thanks for tracking that down, @maxkadel !  I've removed our customization accordingly.